src/Byng/Migration/WebsiteTranslation.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace Byng\Pimcore\Migration;
+
+use Pimcore\Model\Translation;
+use Pimcore\Tool;
+use Pimcore\Model\Translation\Website as SiteTranslation;
+
+/**
+ * Description of SharedTranslation
+ *
+ * @author Asim Liaquat <asim@byng.co>
+ */
+class WebsiteTranslation
+{
+    
+    /**
+     * Add new translations
+     * 
+     * The method accepts an array of translations where the key is the
+     * translation key and the value is an array where each row is a translation
+     * for a specific language. e.g.
+     * 
+     *  $translations = [
+     *      "btn_continue" => [
+     *          "en" => "Continue"
+     *      ],
+     *   ];
+     * 
+     * @param array $translations
+     */
+    public function create(array $translations = [])
+    {
+        Translation\Website::clearDependentCache();
```

Comment:
you've imported Translation\Website above as SiteTranslation, so you could just use that?